### DISCUSSION TITLE
SHARPy can't read the GustVelocityField settings

### DISCUSSION THREAD

**Discussion by elfickuuck** [ImperialCollegeLondon/sharpy#138] [Q&A] [answered]
Hi, 

When I tried to generate a gust velocity field in ['StepUvlm'] solver, SHARPy cannot read my settings. But when I switched to 'SteadyVelocityField' it works. 

<img width="748" alt="Screenshot 2021-04-10 at 4 20 05 PM" src="https://user-images.githubusercontent.com/82260212/114263577-fbd70f00-9a18-11eb-99b8-260d2da751ff.png">
<img width="666" alt="Screenshot 2021-04-10 at 4 10 16 PM" src="https://user-images.githubusercontent.com/82260212/114263586-114c3900-9a19-11eb-94af-dd001ec3a6ab.png">

**Comment by ngoiz** [comment, marked as answer]
Hi,

That looks like the old way we had the gust settings defined. They are now input to the aero solvers in a slightly different way since there are common settings to all gusts and then some particular ones. They are now given as:

```
aero_settings = {'<some_aero_settings>': '<some_aero_settings>',
                  'velocity_field_generator': 'GustVelocityField',
                  'velocity_field_input': {'u_inf': 1,
                                           'gust_shape': '<desired_gust>',
                                           'gust_parameters': '<gust_settings>'}}
```

More detailed info in:
https://ic-sharpy.readthedocs.io/en/master/includes/generators/gustvelocityfield/index.html

Hope this helps

**Comment by ArturoMS13** [comment]
Dear elfickuuck,

Did you copy those settings from the code or the documentation? 
Can you tell us where from so we can update the file to the latest version?

Thank you very much
